src/index.html

```diff
@@ -3,11 +3,11 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Leaderboard</title>
+    <title>The Pirate Leaderboard</title>
 </head>
 <body>
     <div class="container">
-        <header><h1>Leaderboard</h1></header>
+        <header><h1>The Pirate Leaderboard</h1></header>
```

Comment:
Great job with your project! It's looking absolutely awesome! 😄👏 I just wanted to share a couple of small optional suggestions to make it even more fantastic:

- **[OPTIONAL]** Have you thought about automatically refreshing the leaderboard whenever a user adds a new score? This could really boost the user experience of your application without the need for them to click the refresh button! 🔄

- **[OPTIONAL]** Another idea could be to display the scores in descending order, with the highest scores at the top. This way, users can see the best scores right away! 📈🥇

   Keep up the fantastic work, and feel free to consider these ideas! You're doing great! 💪😊